Cargo.toml

```diff
@@ -11,15 +11,13 @@ version = "0.5.0"
 
 [dependencies]
 chrono = "0.4"
-curl = ">= 0.3.0, < 0.5.0"
+futures = "0.1"
```

Comment:
I don't think we need this. Although it would be good to make this crate support futures... perhaps the async/await changes will make it easier.

Reply:
Fixed.